<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>47</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="bt_select_run_dir">
       <property name="toolTip">
        <string>Select a Run directory</string>
       </property>
       <property name="text">
        <string>Select Run Directory</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_run_directory"/>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Train Id</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sb_train_id">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Select TrainId</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Pulse#:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sb_pulse_id">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_pulse">
       <property name="text">
        <string>Pulse#</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_sum">
       <property name="text">
        <string>Sum</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_mean">
       <property name="text">
        <string>Mean</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
